src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -57,6 +57,15 @@ internal TagHelperExecutionContext(string tagName)
         }
 
         /// <summary>
+        /// Indicates whether or not the tag is self closing.
+        /// </summary>
+        public bool SelfClosing
```

Comment:
My comment also applies tothings like `ExecuteChildContentAsync` we probably shoudl have made a Run overload on the `TagHelperRunner` that handled that.